examples/custom_generic_nl_family_custom_types.rs

```diff
@@ -0,0 +1,138 @@
+//! This examples shows you how you can use neli to communicate with a custom kernel module
+//! via a custom (generic) netlink family. It shows you how you can define application-specific
+//! netlink commands and netlink attributes. Generic Netlink commands (the cmd field in the
+//! Generic Netlink header) is just an u8 integer, but for convenience we give it a semantic name.
+//! More information in Linux source code:
+//! https://elixir.bootlin.com/linux/v5.9.12/source/include/uapi/linux/genetlink.h#L14
+//!
+//! THIS EXAMPLE WILL NOT WORK ON YOUR SYSTEM UNLESS YOU CODE AND LOAD A CUSTOM KERNEL MODULE
+//! THAT IMPLEMENTS THIS BASIC (GENERIC) NETLINK FUNCTIONALITY. SEE ALSO HERE:
+//! https://github.com/phip1611/generic-netlink-user-kernel-rust
+
+use neli::consts::nl::Nlmsg;
+use neli::consts::nl::{NlmF, NlmFFlags};
+use neli::consts::socket::NlFamily;
+use neli::genl::{Genlmsghdr, Nlattr};
+use neli::nl::{NlPayload, Nlmsghdr};
+use neli::socket::NlSocketHandle;
+use neli::types::{Buffer, GenlBuffer};
+use neli::utils::U32Bitmask;
+use neli::Nl;
+use std::process;
```

Comment:
Could you switch this to nested imports? I've been meaning to put out some formal coding style guidelines but haven't gotten around to that quite yet.